book.json

```diff
@@ -5,7 +5,7 @@
         "readme": "Introduction.md"
       },
     "catalog": "all",
-    "ignores": ["_book", ".git"],
+    "ignores": ["_book", "node_modules", "Introduction.md"],
```

Comment:
This was an effort to get `gitbook-summary` to ignore that file since it was replacing the `README` as the intro. This config property only works for directories, so `Introduction.md` still shows up.